Update docs to add encoding header to be excluded (#489)
trinodb/trino-gateway: docs/routing-rules.md

        urlPath: https://router.example.com/gateway-rules # replace with your own API path
        excludeHeaders:
            - 'Authorization'
            - 'Accept-Encoding'
```

* Redirect URLs are not supported
* Optionally add headers to the `excludeHeaders` list to exclude requests with corresponding header values
  from being sent in the POST request.

> * Check headers that need to be excluded when making API requests, depending on network configuration.

Reviewer: ```suggestion
* Check headers to exclude when making API requests, specifics depend on the network configuration.
```
